index.html

```diff
@@ -10,229 +10,233 @@
+                            <li>CSS</li>
+                            <li>Ruby on rails</li>
+                        </ul>
+                        <a class="button" href="">See project</a>
+                    </section>
+                </section>
+            </div>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Javascript</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
         </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Javascript</li>
-                    <li>Ruby on rails</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
-        </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Ruby on rails</li>
-                </ul>
-                <a class="button" href="">See project</a>
+        <section id="about">
+            <section class="aboutme">
+                <h1>About me</h1>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.</p>
+                <a class="button" href="">Get My Resume</a>
             </section>
-        </section>
-
-    </section>
 
-
-    <section class="about">
-        <section class="aboutme">
-            <h1>About me</h1>
-            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don't hestiate to contact me.</p>
-             <a class="button" href="">Get My Resume</a>
+            <ul class="skills">
```

Comment:
- [ ] You also added much space between the about section and the skills. I will suggest you reduce the space.

Your Design | Figma Design |
:---------------:|:-------------------:|
![e8](https://user-images.githubusercontent.com/39523312/149405058-244ca94d-f849-4e29-9072-4908c8489e3b.png) |![e7](https://user-images.githubusercontent.com/39523312/149405065-2c53123a-4ef6-4b44-b191-ca1c6c3fef0c.png)

Reply:
Again, intentional design, it looks more _aesthetic_ from my point of view.